distributed/tests/test_gpu_metrics.py

```diff
@@ -6,7 +6,7 @@
 
 @gen_cluster()
 async def test_gpu_metrics(s, a, b):
-    from distributed.diagnostics.nvml import handles
+    from distributed.diagnostics.nvml import _pynvml_handles as handles
 
     assert "gpu" in a.metrics
     assert len(s.workers[a.address].metrics["gpu"]["memory-used"]) == len(handles)
```

Comment:
When running this, I encounter another issue. Short snippet from the failure included below for brevity. Not sure if this is an indication of a bug or another issue in the test.

```python
    @gen_cluster()
    async def test_gpu_metrics(s, a, b):
        from distributed.diagnostics.nvml import _pynvml_handles as handles
    
        assert "gpu" in a.metrics
>       assert len(s.workers[a.address].metrics["gpu"]["memory-used"]) == len(handles)
E       TypeError: object of type 'int' has no len()
```

Reply:
We only return one handle from `_pynvml_handles` as well. So I'm not sure if `len` was suppose to be here or if this is a typo.

https://github.com/dask/distributed/blob/a1dc5f437b39c1b35a9b05cbc048e3a793b89715/distributed/diagnostics/nvml.py#L29-L30

Reply:
It looks like this function use to return a `list`, but PR ( https://github.com/dask/distributed/pull/3810 ) changed it to return a single value. So maybe this test doesn't make sense (at least how it is written) any more.

Reply:
I changed the way nvml/handles work -- we are now only handing back one handle per worker